services/discordService.js

```diff
@@ -68,9 +68,28 @@ const removeRoleFromUser = async (roleId, discordId) => {
   }
 };
 
+const setUserDiscordNickname = async (userName, discordId) => {
```

Comment:
DISCORD_BASE_URL environment variable is not defined. This means that the fetch() function will not be able to connect to the Discord API.

To fix the bug, you need to define the DISCORD_BASE_URL environment variable. You can do this by adding the following line to your .env file:

Reply:
I am accessing it from the `services.discordBot.baseUrl` in the **config/local.js** file.